IngressMaxFields.user.js

```diff
@@ -45,21 +47,31 @@ function wrapper() {
 	self.portalInPolygon = function portalInPolygon(polygon, portal) {
 		var poly = polygon.getLatLngs();
 		var pt = portal.getLatLng();
-
-		for (var c = false, i = -1, l = poly.length, j = l - 1; ++i < l; j = i) {
+		var c = false;
+		for (var i = -1, l = poly.length, j = l - 1; ++i < l; j = i) {
 			((poly[i].lat <= pt.lat && pt.lat < poly[j].lat) || (poly[j].lat <= pt.lat && pt.lat < poly[i].lat)) && (pt.lng < (poly[j].lng - poly[i].lng) * (pt.lat - poly[i].lat) / (poly[j].lat - poly[i].lat) + poly[i].lng) && (c = !c);
 		}
 		return c;
-	}
+	};
 
 	// return if the portal is within the drawtool objects.
 	// Polygon and circles are available, and circles are implemented
 	// as round polygons.
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["L" is not defined.](https://www.codacy.com/app/stefan-nygren/IITC-Ingress-Maxfields-Exporter/file/1765053557/issues/source?bid=3039347&fileBranchId=3045939#l61)